assets/chat/css/generify.scss

```diff
@@ -62,6 +62,57 @@
     }
 }
 
+@keyframes peek {
+    0% {
+        transform: translate(-110%, 0);
+    }
+    
+    50% {
+        transform: translate(-33%, 0);
+    }
+
+    100% {
+        transform: translate(-110%, 0);
+    }
+}
+@keyframes reverse-peek {
+    0% {
+        transform: translate(110%, 0);
+    }
+    
+    50% {
+        transform: translate(33%, 0);
+    }
+
+    100% {
+        transform: translate(110%, 0);
+    }
+}
+
+.generify-peek {
+    clip-path: polygon(0 -100%, 100% -100%, 100% 200%, 0 200%);
+
+    & > .generify-container {
+        animation: peek 6s ease-in-out 4;
```

Comment:
total animation duration should be ~6 seconds and it shouldn't end with an empty space if animateforever is disabled

Reply:
Going to a half step (i.e `1.5`) has the desired effect, where the emote is in frame as the animation ends. I've also modified the keyframe so that the emote comes fully into frame rather than being a little bit offset - this also resolves the issue where if infinite looping is off, the animation "jumps" into frame at the last loop.